src/server/api/controllers/annotations.controller.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const knex = require("../../config/db");
+const error = require("../lib/utils/http-error");
+
+const getAnnotations = async req => {
+  try {
+    return await knex("annotations").select(
+      "annotations.annotation_id",
+      "annotations.title",
+      "annotations.description",
+      "annotations.fk_screenshot_id",
+      "annotations.area",
+      "annotations.updated_at",
+      "annotations.created_at",
+      "annotations.deleted_at"
+    );
+  } catch (error) {
+    return error.message;
+  }
+};
+const getAnnotationsById = async id => {
+  try {
+    const annotations = await knex("annotations")
+      .select(
+        "annotations.id as id",
+        "title",
+        "description",
+        "fk_screenshot_id",
+        "area",
+        "updated_at",
+        "created_at",
+        "deleted_at"
+      )
+      .where({ id: id });
+    if (annotation.length === 0) {
+      throw new error(`encorrect entry with the id of ${id}`, 404);
```

Comment:
Nice work! Fetched screenshot annotations from DB using the updated Swagger. 
Just one small typo!
```suggestion
      throw new error(`incorrect entry with the id of ${id}`, 404);
```

Reply:
sorry I fixed it;